internal/report/report.go

```diff
@@ -82,7 +82,11 @@ type Options struct {
 }
 
 // Generate generates a report as directed by the Report.
-func Generate(w io.Writer, rpt *Report, obj plugin.ObjTool) error {
+func Generate(w io.Writer, rpt *Report, obj plugin.ObjTool, ui plugin.UI) error {
+	if rpt.unexpNegSamples {
+		ui.PrintErr("Profile has negative values. If using -base, consider using -diff_base instead.")
```

Comment:
Sorry, let's do one more tweak to clarify _why_ someone would need to consider using -diff_base instead. How about

`Profile has negative values, percentage values may be incorrect. Profile has negative values. If using -base, consider using -diff_base instead.`

Reply:
Edited the comment, I had a mistake there. The right version is `Profile has negative values, percentage values may be incorrect. If using -base, consider using -diff_base instead.`.

Reply:
Done.